src/app/page/page.component.ts

```diff
@@ -640,6 +640,7 @@ export class PageComponent implements OnInit, OnDestroy {
     });
 
     if (isListener) {
+      this.pageService.addToNavigationStack(pageToNavigateTo.position);
```

Comment:
there is a `page.id` property, and you can find that page from the manifest using `manifest.findPage(pageId)`. It's how pages get referenced in CYOA tools

Reply:
it's also what I included in the urls for cyoa tools

Reply:
So you're using the page ID in the URLs and no longer position?

Reply:
Ah I see. Just saw your Jira ticket update